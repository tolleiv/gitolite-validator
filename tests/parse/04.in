repo @all
    C       =   @devs
    RW+     =   @users
