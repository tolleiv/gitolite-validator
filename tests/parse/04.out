root:
  repo:
    paths:
      @all
    rules:
      rule:
        permission: C
        members:
          @devs
      rule:
        permission: RW+
        members:
          @users
